### PR TITLE
cli

### PR DESCRIPTION
**Pull request by walker1992**
add shell and withdraw cmd for cli

### CODE REVIEW

File: infrastructure/zcli/src/commands.ts [hunk 1]
```diff
@@ -194,6 +194,23 @@ class TxSubmitter {
...
+    private async withdraw(txDetails: TxDetails, fast: boolean) {
+        const { to, token, amount } = txDetails;
+        if (!(await this.syncWallet.isSigningKeySet())) {
+            const changePubkey = await this.syncWallet.setSigningKey({
+                feeToken: token
+            });
+            await changePubkey.awaitReceipt();
+        }
+        const txHandle = await this.syncWallet.withdrawFromSyncToEthereum({
+            ethAddress:to,
```
Comment (lyova-potyomkin) [inline infrastructure/zcli/src/commands.ts:207]: Please use `zk fmt` to format the code.

**Review by lyova-potyomkin — APPROVED** [review]
Nice, thank you!

**Review by walker1992 — COMMENTED** [review]
zk fmt

**Review by popzxc — APPROVED** [review]
Thanks!

**Review by YorkSvit — COMMENTED** [review]
infrastructure/zcli/src/index.ts

**Pull request merged**